deployment/helm/sshd/templates/sshd-deployment.yaml

```diff
@@ -0,0 +1,181 @@
+{{- $image := "linuxserver/openssh-server:latest" -}}
```

Comment:
I wonder if this should be fixed to a certain version so that we are not surprised by an update that is incompatible with our config or our deployment...

Reply:
That may be a good idea, sure.  I'll find the latest one.